pykeybasebot/bot.py

```diff
@@ -69,6 +69,7 @@ def __init__(
         keybase: Optional[str] = None,
         home_path: Optional[str] = None,
         pid_file: Optional[str] = None,
+        disable_typing: Optional[bool] = True,
```

Comment:
```suggestion
        disable_typing: Optional[bool] = True,  # notifications when a user is currently typing a reply
```
or some such way of conveying what this thing is